tests/testthat/test-combinedscatterplot.R

```diff
@@ -582,3 +582,29 @@ test_that("quadrants",
                                    x.midpoint.input = c(2,3), y.midpoint.input = 12),
                    "The input for the x midpoint has multiple elements")
 })
+
+test_that("Missing values and named colors",
```

Comment:
Does this actually test your changes?

Reply:
No, it doesn't (well, it checks it runs without error, but not that it looks correct). I'll add a Standard R test when this is merged